overlays/turnkey.d/security/etc/ssl/turnkey.cnf

```diff
@@ -0,0 +1,52 @@
+#
+# SSL template for TurnKey GNU/Linux appliances
+# Based on the SSLeay example configuration file.
+#
+
+RANDFILE                = /dev/urandom
```

Comment:
IIRC ideally /dev/random should be used instead. /dev/urandom is easier/nicer in some respects as it's non-blocking, but AFAIK /dev/random provides superior pseudo random numbers...